Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -140,7 +140,7 @@ extension BrowserViewController: WKUIDelegate {
                 })
             }
 
-            actions.append(UIAction(title: Strings.ContextMenuOpenInNewPrivateTab, image: UIImage.templateImageNamed("menu-NewPrivateTab"), identifier: UIAction.Identifier("linkContextMenu.openInNewPrivateTab")) { _ in
+            actions.append(UIAction(title: Strings.ContextMenuOpenInNewPrivateTab, image: UIImage.templateImageNamed("menu-ForgotMode"), identifier: UIAction.Identifier("linkContextMenu.openInNewPrivateTab")) { _ in
```

Comment:
it should probably be called `ForgetMode`

Reply:
Done